scripts/travis_lint.sh

```diff
@@ -5,9 +5,15 @@ set -e
 script_folder=`dirname $0`
 pip install --user unidiff
 
+if ! [[ -e $script_folder/cpplint.py ]]
+then
+  echo "CPP Linter script could not be found in the $script_folder directory"
+  echo "Ensure cpplint.py is inside the $script_folder directory then run again"
+  exit 1
+fi
+
 if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
-  $script_folder/run_lint.sh HEAD~1 # Check for errors introduced in last commit
+  $script_folder/run_diff.sh '$script_folder/cpplint.py $file 2>&1 >/dev/null' HEAD~1 # Check for errors introduced in last commit
```

Comment:
cpplint prints all relevant output to stderr. We are not interested in stuff that it prints to stdout.